src/components/projects/Inspector.tsx

```diff
@@ -58,7 +58,7 @@ const Inspector: React.FC<Props> = ({ editor }) => {
     trackModelMap: store.trackModelMap,
     attachments: store.attachments,
     fileManagement: store.fileManagement,
-    editorSelectedComment: store.comment,
+    editorSelectedComment: store.newComments.size,
```

Comment:
This is a bit confusing now. We have a "editorSelectedComment" property, which value now is integer. Would that be possible to rename appropriately?

Reply:
updated